client/Router.jsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
+// import axios from 'axios';
+//import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { BrowserRouter as BrowserRouter, Switch, Route } from 'react-router-dom';
+import { AuthProvider } from './AuthContext';
+import { Container, CssBaseline } from '@material-ui/core';
+import {SignUp} from './screens/Signup';
+import {UseStyles} from './components/UseStyles';
+import {Dashboard} from './screens/Dashboard';
+import {Login} from './screens/Login';
+import {Logout} from './screens/Logout'
 
-export const Router = () => (
-  <div>
-    hello world new changasdasde
-  </div>
-);
+export const Router = () => {
+  const classes = UseStyles();
```

Comment:
Nit: `useStyles` to follow [React Hooks](https://reactjs.org/docs/hooks-intro.html) convention

Reply:
changed. 
![image](https://user-images.githubusercontent.com/35232697/105070655-3a2be600-5a49-11eb-916a-098b7c7c129e.png)

Reply:
the filename should stay as UseStyles.js though right?

Reply:
Usually it's good for filename to match internal casing